docs/api/taxons.rst

```diff
@@ -931,3 +931,62 @@ Exemplary Response
 .. code-block:: text
 
     STATUS: 204 No Content
+
+Set position of product in a Taxon
+----------------------------------
+
+The products in Sylius can by group by taxon, therefore every product has an relation between itself and selected taxon.
+What is more, every product can have specific position in taxon to which it is belongs to. To put products in specific order
```

Comment:
`What is more, every product can have a specific position in the taxon to which it belongs. To put products in a specific order (...)`